AdventOfCode2021/day3_2/day3_2.py

```diff
@@ -0,0 +1,74 @@
+#!python3
+
+from typing import List
+
+def file_read(file_name) -> List[str]:
+    file_data_list: List[str] = []
+    with open(file_name) as f:
+        for line in f:
+            file_data_list.append(line.strip())
+    return file_data_list
+
+
+def list_to_string(list_value: List[str]) -> str:
+    return ("".join([str(value) for value in list_value]))
```

Comment:
If `list_value` is indeed `List[str]`, then you can get the result you need by just `"".join(list_value)`.
The expression `"".join([str(value) for value in list_value])` is required **only** if `list_value` contains `int` elements.

Reply:
Got it!